### DISCUSSION TITLE
Is there an API available? (SillyTavern integration)

### DISCUSSION THREAD

**Discussion by xpgx1** [rupeshs/fastsdcpu#192] [Q&A] [answered]
Greetings! Very nice project =)

I'm curious if you plan on adding cmd_flags as an easy way to custmize this installation - and if you perhaps expose the api, if this is part of your project scope? It would enable me to use this instance of SD while also using a fully engaged (and thus VRAM hungry) LLM via the GPU.

Have a gr8 rest of the weekend, thank you for publishing your project!

**Comment by xpgx1** [comment]
Additionally, how can one disable the black image output? This seems... very _prohibitive_ and excessive. I'm not keen on generating nsfw images (i want portraits, nothing spicey), but such a "safeguard" is entirely unwanted - **if there is no toggle.**

**Reply by rupeshs** [reply]
Uncheck "Use safety checker" setting.
![image](https://github.com/rupeshs/fastsdcpu/assets/3255994/b898281a-c442-4f4d-b182-1555feafc5e7)

**Comment by rupeshs** [comment]
@xpgx1 I'm planning to basic API support in the next release.

**Comment by rupeshs** [comment, marked as answer]
@xpgx1  Added API support in the  v1.0.0-beta.32 release https://github.com/rupeshs/fastsdcpu/releases/tag/v1.0.0-beta.32
